src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -111,12 +111,14 @@
 
                          :currentNetwork              config/default-network
                          :previewPrivacy              config/blank-preview?}]
-    {effect    request
-     :dispatch [:navigate-to :generating-keys]
-     :db       (-> db
-                   (dissoc :multiaccounts/login)
-                   (dissoc :auth-method)
-                   (assoc :onboarding-2/new-account? true))}))
+    {effect          request
+     :dispatch       [:navigate-to :generating-keys]
+     :dispatch-later [{:ms       constants/onboarding-generating-keys-animation-duration-ms
```

Comment:
waits until generate keys animation has finished, and then has a fallback check to ensure keys are definitely generated in which case the transition will be slightly delayed 👍